src/modules/LanItems.rb

```diff
@@ -2018,21 +2027,24 @@ def Commit
         newdev["INTERFACETYPE"] = @type
       end
 
-      NetworkInterfaces.Name = Ops.get_string(@Items, [@current, "ifcfg"], "")
-      NetworkInterfaces.Current = deep_copy(newdev)
+      if GetCurrentMap() != newdev
+        NetworkInterfaces.Name = Items()[@current]["ifcfg"] || ""
+        NetworkInterfaces.Current = deep_copy(newdev)
```

Comment:
just first line is new ;-)

1. checks if new and current sysconfig differs
2. touches ```NetworkInterfaces``` internal state machine to set proper config to update proper configuration. In my POV this should be replaced by ```NetworkInterfaces.Select```. I'll most probably do it in a later commit.
3. updates whole devmap in ```NetworkInterfaces```. In my POV it can be replaced by ```NetworkInterfaces.Change2```. As in above case I'll nost probably do so in a later commit.